### PR TITLE
Activate portfolio with thumbs

### PR DESCRIPTION
**Pull request by francois-blanchard**

### CODE REVIEW

File: utils/create-thumb.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (VictorTpo) [inline utils/create-thumb.sh:1]: What is that?

**Review by VictorTpo — APPROVED** [review]

**Comment by VictorTpo** [comment]
When we do front updates, we should add a screenshot in the PR review ;)

**Comment by francois-blanchard** [comment]
> When we do front updates, we should add a screenshot in the PR review ;)

My bad, I'm not used to it. 😁 

Screenshot 2020-06-01 at 17 40 19

File: utils/create-thumb.sh (hunk 1 above)
Comment (francois-blanchard) [reply utils/create-thumb.sh:1]: When someone will execute this shell script, it's gonna use `/bin/bash` to execute it.

**Pull request merged**